.rubocop_local.yml

```diff
@@ -3,3 +3,14 @@ GlobalVars:
   # Loggers
   - $lenovo_log
   - $log
+
+# Ignores the file at lib folder that needs to have a naming with dashes.
+AllCops:
+  Exclude:
+  - lib/manageiq-providers-lenovo.rb
+
+# Forcing Single-quote string literals except in interpolations
+Style/StringLiterals:
+  Enabled: true
+Style/StringLiteralsInInterpolation:
+  Enabled: true
```

Comment:
Also we explicitly disable these for the MIQ source https://github.com/ManageIQ/guides/blob/master/.rubocop_base.yml#L235-L238 and shouldn't be enabled on a provider repo

Reply:
Ok. I will undo this change on PR #181